scripts/apply-labels-3scale.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# One off script to add extra labels to all resources in 3scale namespace
+THREESCALE_COMPONENT=threescale_component 
+THREESCALE_COMPONENT_ELEMENT=threescale_component_element
+
+oc project 3scale
```

Comment:
Noting in production it will be openshift-3scale, only in RHPDS will it be just 3scale.  Suggest using:
```
oc project openshift-3scale || oc project 3scale
```
That way it works for both consistently.

Reply:
:+1: